### PR TITLE
fix: auth login error codes issue

### PR DESCRIPTION
**Pull request by amitojsingh**
This PR contains the fix for #239 with improved error messages

### CODE REVIEW

**Review by Gauravjeetsingh — APPROVED** [review]
LGTM

File: src/utils/firebaseErrorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import CONSTANTS from 'constants/index';
+const firebaseErrorHandler = (code: string) => {
```
Comment (MySmilingTurban) [inline src/utils/firebaseErrorHandler.ts:2]: nit: add empty line before const

File: src/utils/firebaseErrorHandler.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    'auth/invalid-password': CONSTANTS.ERROR_PWD,
+    'auth/too-many-requests': CONSTANTS.TOO_MANY_REQUESTS,
+    'auth/operation-not-allowed': CONSTANTS.OPERATION_NOT_ALLOWED,
+    'auth/weak-password': CONSTANTS.WEAK_PASSWORD,
+  };
+  return (
+    codeMessagesMap[code] || codeMessagesMap[code] || 'An unexpected authentication error occurred'
+  );
+};
+export default firebaseErrorHandler;
```
Comment (MySmilingTurban) [inline src/utils/firebaseErrorHandler.ts:20]: nit: add empty line before export

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
one minor change, everything else looks good to me!

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**